File: src/main/java/kitchenpos/eatinorders/tobe/infra/persistence/eatinorder/TobeJpaEatInOrderRepository.java
package kitchenpos.eatinorders.tobe.infra.persistence.eatinorder;

import java.util.UUID;
import kitchenpos.eatinorders.tobe.domain.eatinorder.EatInOrder;
import kitchenpos.eatinorders.tobe.domain.eatinorder.TobeEatInOrderRepository;
import org.springframework.data.jpa.repository.JpaRepository;

public interface TobeJpaEatInOrderRepository extends TobeEatInOrderRepository,
    JpaRepository<EatInOrder, UUID> {

}

File: src/main/kotlin/kitchenpos/order/tobe/eatinorder/infra/TobeJpaEatInOrderRepository.kt
package kitchenpos.order.tobe.eatinorder.infra

import java.util.*
import kitchenpos.order.tobe.eatinorder.domain.EatInOrder
import kitchenpos.order.tobe.eatinorder.domain.EatInOrderRepository
import org.springframework.data.jpa.repository.JpaRepository

interface TobeJpaEatInOrderRepository : EatInOrderRepository, JpaRepository<EatInOrder, UUID>
